x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/support/Automatons.java

```diff
@@ -25,9 +27,15 @@
 
 public final class Automatons {
 
+    public static final Setting<Integer> MAX_DETERMINIZED_STATES_SETTING =
+        Setting.intSetting("xpack.security.automata.max_determinized_states", 100000, DEFAULT_MAX_DETERMINIZED_STATES,
+            Setting.Property.NodeScope);
```

Comment:
Those arguments confued me at first because a constant named `DEFAULT_...` is being used to set the minimum (3rd arg), and the numeric literal is used for the `defaultValue` (2nd arg).

It's not a big deal, but I suspec it will confuse others down the track.